src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -88,15 +88,21 @@ const MultiSelectMenuButton = React.forwardRef<
           <Icon
             align="right"
             id={`${multiSelectId}-selected-items-count-icon`}
+            marginLeft="xs"
             name="close"
             size="xsmall"
           />
         </Box>
       )}
-      <Box as="span" pr="s">
+      <Box as="span" title={multiSelectLabel} __css={styles.buttonLabel}>
```

Comment:
Why is the `title` attribute set here? Is it for accessibility purposes? Should it use `aria-label` instead?

Reply:
The was introduced to help users when the `MultiSelect` label is truncated.  The truncation is purely handled by styles, so there should be no need for an aria-label with the same thing.  The full text is in there on the button.  It is just not all displayed in the browser for certain situations.